packages/nuxeo-retention-package/src/main/resources/install/templates/retention/nxserver/config/retention-compliance-config.xml.nxftl

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+
+<component name="org.nuxeo.retention.compliance.types">
```

Comment:
Please remove the blank line before `<component` as well.

Reply:
Can remove 👍 
But, just to clarify, it is consistent with the examples provided 😉